### PR TITLE
Hubv2 2024

### PR DESCRIPTION
**Pull request by LGO44**
This  PR is  based on your version 1.5.5 and it could replace my previous PR Hubv2lgo #6 
It works with both first and second generation of Wiser Hub

Many simplifications in the code

May be we can close the previous PR

### CODE REVIEW

**Comment by LGO44** [comment]
@msp1974 , what do you think about this PR? I've added some properties to improve the use of new features ( energy and summer comfort...)

**Comment by msp1974** [comment]
Sorry just getting round to looking at this.  Can you send me a diagnostic file to test it with.  thx

File: aioWiserHeatAPI/heating_actuator.py [hunk 1]
```diff
@@ -172,52 +173,6 @@ def output_type(self) -> str:
         """Get output type"""
         return self._device_type_data.get("OutputType", TEXT_UNKNOWN)
 
-    # added by LGO
```
Comment (msp1974) [inline aioWiserHeatAPI/heating_actuator.py:175]: What is the reason for removing all these?  Aren't they needed for the v1 hub as it doesn't have the equipment dataset?

Comment (msp1974) [reply aioWiserHeatAPI/heating_actuator.py:175]: OK, don't worry - got it now.  Just having to get back up to speed on v2 hub.  These are in equipment?

**Pull request merged**